README.md

```diff
@@ -23,10 +22,13 @@ Example Playbook
 
     - hosts: servers
       roles:
-         -
-           role: devops_cmp.ansible_filebeat
-           version: 6.5.4
-           log_tags: ['prod', 'myapp', 'mycomponent']
+         - role: devops_cmp.ansible_filebeat
+           version: 7.5.0
```

Comment:
processors:
   - paths: ['/var/log/httpd/*access.log']
      tags: ['httpd', 'access', 'isprime', '2.2']
      type: 'log'
   - paths: ['/var/log/httpd/*error.log', 'error_log']
      tags: ['httpd', 'error', '2.2']
      type: 'log'
   - host: 'localhost:5514'
      tags: ['httpd', 'error', '2.2']
      type: 'udp'

Reply:
what about something like this?

Reply:
yeah. Much easier to understand